### ISSUE TITLE
Pretrained OSCAR Model

### ISSUE THREAD

**Issue by dapurv5** [Alcoholrithm/Oscar_Scripts#1]
Hi,
I was trying to get the image captioning code running from your repository. I was wondering if you can point me to the location of the 'checkpoint-59-554820' model that you use

**Comment by Alcoholrithm** [comment]
Hi, dapurv5
Unfortunately, I didn't upload pretrained model yet.
But if you leave an email address in the comment, I'll send you the model as soon as possible.

**Comment by dapurv5** [comment]
Hi,
Thanks a ton, would really appreciate that. My email is dapurv5@gmail.com

**Comment by Alcoholrithm** [comment]
I just send you an email now.
If there is any problem, please leave a comment.

**Comment by dapurv5** [comment]
Hi,
Thank you for sharing the model. I will try with this model you shared.
Might I ask how you trained the model?

Also, is it possible for you to include a public LICENSE (like MIT) and
reference in your code. I am still playing around with it but in the future
if I use your code, it would be good to cite your code.

--
Regards,
Apurv Verma

**Comment by Alcoholrithm** [comment]
The dataset that I used is MS-COCO 2017 and I preprocessed appropriately for Oscar.
I finetuned the pretrained large model which the Oscar repo's VIinVL_MODEL_ZOO.md provided according to the script that also in Oscar repo's VIinVL_MODEL_ZOO.md. 
But, there are some problems with run_captioning.py, so I modified the code a little bit and then finetuned the model.

Lastly, I'll post the license and reference this weekend.
